assets/stylesheets/_components.scss

```diff
@@ -318,6 +318,7 @@
 @import 'notices/style';
 @import 'my-sites/domains/domain-management/style';
 @import 'my-sites/domains/domain-management/email/style';
+@import 'my-sites/email/email-forwarding/style';
```

Comment:
This should be included via the JS file. See:  https://github.com/Automattic/wp-calypso/issues/27515

Reply:
👍 will do, thanks for pointing that out